docs/topics/native/native-target-support.md

```diff
@@ -0,0 +1,100 @@
+[//]: # (title: Kotlin/Native target support)
+
+The Kotlin/Native compiler supports a great number of different targets, though it is hard to provide the same level of
+support for all of them. This document describes which targets Kotlin/Native supports and breaks them into several
+tiers depending on how well the compiler supports them.
+
+> We can adjust the number of tiers, the list of supported targets, and their features as we go.
+> 
+{type="note"}
+
+Mind the following terms used in tier tables:
+
+* **Gradle target name** is a [target preset](multiplatform-set-up-targets.md) that is used in the
+    Kotlin Multiplatform Gradle plugin to enable the target.
+* **Target triple** is a target name according to the `<architecture>-<vendor>-<system>-<abi>` structure that is commonly
+  used by [compilers](https://clang.llvm.org/docs/CrossCompilation.html#target-triple).
+* **Running tests** indicates out of the box support for running tests in Gradle and IDE.
+  
+    This is only available on a native host for the specific target. For example, you can run `macosX64` and `iosX64` tests
+    only on macOS x86-64 host.
+
+## Tier 1
+
+* The target is regularly tested on CI to be able to compile and run.
+* We're doing our best to provide a source and [binary compatibility between compiler releases](https://youtrack.jetbrains.com/issue/KT-42293).
+
+| Gradle target name     | Target triple                 | Running tests | Description                                    |
+|------------------------|-------------------------------|---------------|------------------------------------------------|
+| `linuxX64`             | `x86_64-unknown-linux-gnu`    | ✅             | Linux on x86_64 platforms                      |
```

Comment:
Each (except mingwX64) description cell contains "platforms" word. Maybe we can omit it?

Reply:
I'd rather keep it as it makes target details more descriptive